client/Combatant/CombatantDetails.tsx

```diff
@@ -43,6 +43,7 @@ export class CombatantDetails extends React.Component<CombatantDetailsProps> {
 
     return (
       <div className="c-combatant-details">
+        <hr />
```

Comment:
Currently, there is an `<hr />` element rendered at the end of the StatBlock before the Persistent Notes, but you're right to notice that if the statblock isn't completely scrolled, you don't see it in the right spot.

This doesn't look like the correct spot; this would put a new separator at the top of the statblock. I think you should look at removing the `<hr />` from the end of the StatBlock component, and putting it at the top of the `renderedNotes` node.

Reply:
It is also related to the Description. If the statblock has a description then the separator from the `StatBlock` component would be before that.

Do you still recommend moving it from `StatBlock` to `renderedNotes`?